src/ccnl-core/include/ccnl-defs.h

```diff
@@ -71,12 +71,20 @@
 # define CCNL_DEFAULT_MAX_PIT_ENTRIES    (-1)
 #endif
 
-#define CCNL_CONTENT_TIMEOUT            300 // sec
-#define CCNL_INTEREST_TIMEOUT           10  // sec
-#define CCNL_MAX_INTEREST_RETRANSMIT    7
+#ifndef CCNL_CONTENT_TIMEOUT
+# define CCNL_CONTENT_TIMEOUT            300 // sec
```

Comment:
While you're at it, could you please make the constants unsigned, e.g. 
``` C
#ifndef CCNL_CONTENT_TIMEOUT
#define CCNL_CONTENT_TIMEOUT       (300u)
#endif
```
And I don't want to sound too pesky, but could you also provide documentation (good old boy-scout rule)?

Reply:
@mfrey thanks for the fast review. Changing the signedness of these defines is a bigger task which is out of scope of this PR. All over the code you get build errors (`comparison between signed and unsigned integer`) and I really don't wanna change core data structures here :/.
Documentation seems reasonable but I tend to do this in separate PR as well. What do you think?

Reply:
> thanks for the fast review. Changing the signedness of these defines is a bigger task which is out of scope of this PR.

I have to admit that I probably was to naive. 

> Documentation seems reasonable but I tend to do this in separate PR as well.

Looking forward for your PR on documentation!  There are so many "mysterious" lines of code in ccn-lite I'm always happy if I can read some documentation (also applies to obvious things such as "defines").